### DISCUSSION TITLE
Network Security Testing

### DISCUSSION THREAD

**Discussion by qualy-sec** [community/community#144651] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Security testing guarantees that the system is safe from malware, attackers, and other online threats. To attain this level of protection, the system examines all security-related requirements. [Network security testing](https://qualysec.com/network-security-testing/) is a procedure that identifies security vulnerabilities and flaws in a network. It can encompass several network devices and applications and direct communication with customers, other organizations, and government agencies.

**Comment by mecodeatlas** [comment]
Hi @qualy-sec, we’re glad you’re here 😀

Thanks for sharing your website with the community. Unfortunately, we currently do not allow self-promotion or advertising on the Community Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
